explorer/explorer_pods.go

```diff
@@ -40,6 +69,22 @@ func (n *PodsExplorer) List() error {
 	return nil
 }
 
+// gets pods from map of owners
```

Comment:
function comments should start with the function name

```
// getOwnerPods is a function that $reson-for-function
func getOwnerPods(){
}
```

https://blog.golang.org/godoc-documenting-go-code